actions/Create DB User/code.js

```diff
@@ -0,0 +1,26 @@
+const fetch = require('node-fetch')
+
+exports.onExecutePostLogin = async (event, api) => {
+  // 1.  
+  const SECRET = event.secrets.AUTH0_HOOK_SECRET
+  
+  // 2.
+  if (event.user.app_metadata.localUserCreated) {
+    return
+  }
+
+  // 3.
+  const email = event.user.email
+
+  const url = 'NGROK_URL_REPLACE_HERE'
+  // 4.
+  const request = await fetch(url+'/api/auth/hook', { 
```

Comment:
This code look to support template literals https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

It would be nicer to write:

```diff
-  const request = await fetch(url+'/api/auth/hook', { 
+  const request = await fetch(`${url}/api/auth/hook`, { 
```

Reply:
In the futur, if the fetch fails, we should add `try/catch` block and logging